iceoryx_hoofs/include/iceoryx_hoofs/internal/design_pattern/polymorphic_handler.inl

```diff
@@ -20,6 +20,7 @@
 #include "iceoryx_hoofs/design_pattern/polymorphic_handler.hpp"
 #include "iceoryx_hoofs/design_pattern/static_lifetime_guard.hpp"
 #include <atomic>
+#include <exception>
```

Comment:
Aaaargh. This is required for `std::terminate`, right?

Reply:
Yes, changed to abort.